File: src/main/java/io/bloco/faker/components/SlackEmoji.java
package io.bloco.faker.components;

import io.bloco.faker.FakerComponent;
import io.bloco.faker.FakerData;

public class SlackEmoji extends FakerComponent {

    public SlackEmoji(FakerData data) {
        super(data);
    }

    public String people() {
        return fetch("slack_emoji.people");
    }

    public String nature() {
        return fetch("slack_emoji.nature");
    }

    public String foodAndDrink() {
        return fetch("slack_emoji.food_and_drink");
    }

    public String celebration() {
        return fetch("slack_emoji.celebration");
    }

    public String activity() {
        return fetch("slack_emoji.activity");
    }

    public String travelAndPlaces() {
        return fetch("slack_emoji.travel_and_places");
    }

    public String objectsAndSymbols() {
        return fetch("slack_emoji.objects_and_symbols");
    }

    public String custom() {
        return fetch("slack_emoji.custom");
    }

    public String emoji() {
        return parse(fetch("slack_emoji.emoji"));
    }
}

File: src/main/kotlin/io/bloco/faker/components/SlackEmoji.kt
package io.bloco.faker.components

import io.bloco.faker.FakerComponent
import io.bloco.faker.FakerData

class SlackEmoji(data: FakerData) : FakerComponent(data) {
    fun people(): String {
        return fetch("slack_emoji.people")
    }

    fun nature(): String {
        return fetch("slack_emoji.nature")
    }

    fun foodAndDrink(): String {
        return fetch("slack_emoji.food_and_drink")
    }

    fun celebration(): String {
        return fetch("slack_emoji.celebration")
    }

    fun activity(): String {
        return fetch("slack_emoji.activity")
    }

    fun travelAndPlaces(): String {
        return fetch("slack_emoji.travel_and_places")
    }

    fun objectsAndSymbols(): String {
        return fetch("slack_emoji.objects_and_symbols")
    }

    fun custom(): String {
        return fetch("slack_emoji.custom")
    }

    fun emoji(): String {
        return parse(fetch("slack_emoji.emoji"))
    }
}
